contracts/finance/VestingWalletCliff.sol

```diff
@@ -46,6 +46,6 @@ abstract contract VestingWalletCliff is VestingWallet {
         uint256 totalAllocation,
         uint64 timestamp
     ) internal view virtual override returns (uint256) {
-        return timestamp < cliff() ? 0 : super._vestingSchedule(totalAllocation, timestamp);
+        return timestamp <= cliff() ? 0 : super._vestingSchedule(totalAllocation, timestamp);
```

Comment:
I don't think this change is a big deal, but I also think its somehow wrong.

IMO the cliff is the moment where things start to unlock. which means you return zero until `cliff - 1`.
Anyway, in both cases, its just a mater of waiting one more second, so I don't see how this is even an issue.